packages/admin/resources/lang/pl/resources/pages/create-record.php

```diff
@@ -2,9 +2,9 @@
 
 return [
 
-    'title' => 'Utwórz :label',
```

Comment:
I used "Utwórz" because it is in many popular applications, e.g. in Wordpress it has "Dodaj wpis" and not "Dodawanie Wpisów".